pages/national_resources/ch_resources.md

```diff
@@ -173,8 +173,7 @@ The ETH Domain falls under the authority of the Federal Department of Economic A
 [Swiss Open Research Data](https://www.swissuniversities.ch/en/topics/digitalisation/open-research-data/national-strategy-and-action-plan)
 In 2021, the State Secretariat for Education, Research and Innovation (SERI), the Swiss Conference of Rectors of Higher Education Institutions (swissuniversities), the Swiss National Science Foundation (SNSF) and Switzerland’s two federal institutes of technology (ETHZ in Zurich and EPFL in Lausanne) signed an agreement to develop a national strategy on open research data.
 
-The [Swiss ORD Strategy](https://www.swissuniversities.ch/en/topics/digitalisation/open-research-data/swiss-open-research-data-grants) provides a framework for the development of practices
-built around sharing research data in Switzerland, and for governing the services and infrastructures that support researchers and enable ORD practices. It should enable Switzerland to implement a coherent and ambitious open science policy in line with European and global developments.
+The [Swiss ORD Strategy](https://www.swissuniversities.ch/en/topics/open-science/open-science-programme/calls) provides a framework for the development of practices built around sharing research data in Switzerland, and for governing the services and infrastructures that support researchers and enable ORD practices. It should enable Switzerland to implement a coherent and ambitious open science policy in line with European and global developments.
```

Comment:
This is a link to upcoming open calls, does not describe ORD strategy which seems to be mentioned here https://openresearchdata.swiss/what-is-ord/

Reply:
```suggestion
The [Swiss ORD Strategy]( https://openresearchdata.swiss/what-is-ord/) provides a framework for the development of practices built around sharing research data in Switzerland, and for governing the services and infrastructures that support researchers and enable ORD practices. It should enable Switzerland to implement a coherent and ambitious open science policy in line with European and global developments.
```